script.js

```diff
@@ -1,12 +1,116 @@
-//You can edit ALL of the code here
+//TV Show Project
+
+const allEpisodes = getAllEpisodes();
```

Comment:
It was a good idea to move this to a global scope as we need to access this function more than once and outside of setup. Well done.